### ISSUE TITLE
Tray icon broken on Ubuntu 22.04

### ISSUE THREAD

**Issue by tinkermakar** [yuezk/GlobalProtect-openconnect#354]
**Describe the bug**
Tray icon broken on Ubuntu 22.04

**Expected behavior**
be green when connected and load whatever it's trying to load when hovered

**Screenshots**
![image](https://github.com/yuezk/GlobalProtect-openconnect/assets/68788448/f7c1a07f-5728-4fc4-b95d-2260f3d9a0c7)

**Environment:**
 - OS: Ubuntu 22.04
 - Desktop Environment: GNOME
 - Is remote SSH? No

**Comment by yuezk** [comment]
@tinkermakar do you checked this?

<img width="765" alt="image" src="https://github.com/yuezk/GlobalProtect-openconnect/assets/3297602/a4380f78-7bc6-45d6-a159-940e01a5f926">

**Comment by dhsmith1001** [comment]
I'm seeing this issue after upgrading to version 2.3.4-ppa1:

apt search globalprotect-openconnect
globalprotect-openconnect/mantic,now 2.3.4-ppa1~ubuntu23.10 amd64 [installed]

The tray icon does not appear. When I close the GUI window the process stays running (visible in ps output), but no tray icon.

**Comment by yuezk** [comment]
@dhsmith1001 do you mean that the tray icon never appears? Or does it appear at first but disappear after some operation?

**Comment by dhsmith1001** [comment]
> @dhsmith1001 do you mean that the tray icon never appears? Or does it appear at first but disappear after some operation?

Yes, it never appears. However, after a few reboots it started appearing again.
